README.md

```diff
@@ -145,3 +145,28 @@ Pretty-printed JSON metadata listing
 Query offset(s) by timestamp(s)
 
     $ kafkacat -b mybroker -Q -t mytopic:3:2389238523  mytopic2:0:18921841
+
+# Running in docker
+
+There is no official image build yet but you may run `docker build -t kafkacat .` or use for example `solsson/kafkacat@sha256:3075a6ca2d8431910cf01af13acd2ef9542c8ed48547733eb5c4321364ef0b66`.
```

Comment:
It is better not to mention a specific docker image since it will become outdated soon enough.